code/modules/client/client procs.dm

```diff
@@ -436,6 +436,8 @@
 		on_holder_add()
 		add_admin_verbs()
 		admin_memo_output("Show", 0, 1)
+		if(custom_event_admin_msg && custom_event_admin_msg != "")
```

Comment:
Keep in mind this is going to show to mentors, too - they also have a `holder` datum